survey.html

```diff
@@ -0,0 +1,77 @@
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <div class="container">
+  <header class="header">
+    <h1 id="title" class="text-center"><font color="green">Survey Form</font></h1>
+    <p id="description" class="description text-center" style="background-color:black";><font color="#7FFF00">
+      Thank you for taking the time to help us improve the platform
+  </font></p>
+  </header>
+
+    <form id="survey-form">
+      <div class="form-group">
+      <label id="name-label" for="name">Name</label><br>
+      <input type="text" id="name" placeholder="Enter your name" required /> <br> 
+      <div class="form-group">
+      <label id="email-label" for="email">Email</label><br>
+      <input type="email" id="email" placeholder="Enter your Email" required /> <br>
+        <div class="form-group">
+      <label id="number-label" for="number"
+        >Age<span class="clue">(optional)</span></label
+      >
+      <input
+        type="number" name="age" id="number" min="10" max="99"
+        class="form-control"
+        placeholder="Age"
+      />
+    </div>
+      <label>Select County</label> <br>
+      <select id="dropdown" >
+         <option value="usa">USA</option>
+         <option value="UK">UK</option>
+         <option value="India">India</option>
+         <option value="Nigeria">Nigeria</option>
+      </select> <br> 
+      
+      <label>Select two or more programming languages you can work with</label> <br>
+       Html/Css <input value="Html Css" type="checkbox" multiple name="skill"> <br>
+       Javascript <input value="Javascript" type="checkbox" multiple name="skill"> <br>
+       Csharp <input value="Csharp"  type="checkbox" multiple name="skill"> <br>
+       Python <input value="Python" type="checkbox" multiple name="skill"> <br>
+       C++ <input value="C++" type="checkbox" multiple name="skill"> <br>
+       PHP <input value="Php"  type="checkbox" multiple name="skill"> <br>
+       Java <input value="Java" type="checkbox" multiple name="skill"> <br>
+       C <input value="C" type="checkbox" multiple name="skill"> <br>
+
+
+      <label>What are your skills</label> <br>
+       Web design <input value="Web design" type="radio" multiple name="skill"> <br>
+       Web Development <input value="Web Development" type="radio" multiple name="skill"> <br>
+       Graphic Design <input value="Graphic Design"  type="radio" multiple name="skill"> <br>
+       UI/UX <input value="ui/ux" type="radio" multiple name="skill"> <br>
+
+      <div class="form-group">
+      <label> Any comments or suggestions?</label><br>
+      <textarea
+        id="comments"
+        class="input-textarea"
+        name="comment"
+        placeholder="Enter your comment here...."
+      ></textarea>
+    </div>
+     <div class="form-group">
+      <button type="submit" id="submit" class="submit-button">
+        Submit
+      </button>
+    </div>
+
+    </form>
```

Comment:
- [ ] [required] I would suggest you to use semantic HTML tags, that will help you with SEO (Search Engine Optimization) => So wrap this section inside the `<main></main> ` tag

Reply:
Nice to see that you've used inline comments to tell where students need to add the code.